Automattic/jetpack
# Beta plugin: Detect when master needs an update

#### Changes proposed in this Pull Request:

We detected when a PR has been merged and needs updating to master, and
when a PR has an update, and when an RC has an update. But we forgot to
check for when master has been updated.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Use JPBeta to install a master version of a plugin (e.g. Jetpack), then wait for a master merge. Or manually adjust the metadata in the plugin's `.jpinfo.json` file (e.g. `/path/to/wp-content/plugins/jetpack-dev/.jpinfo.json`) to reflect an older version.
* Visit the JPBeta settings page. See if it detects that an update to the plugin is needed.
